content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -60,8 +60,8 @@ Login.gov is a secure, government website that adheres to the highest standards
     * Click “Continue”
 1.  Enter your social security number and click “Continue”
 1.  Verify the information you have submitted so far is correct and click “Continue”
-1.  Enter a phone number with your name on the plan. This is the last step for identity verification.
-    * If you do not have a phone plan with your name on it or a U.S. phone number, click on “Get a code via mail instead.” You will have to wait to receive a letter in the mail and then follow the instructions to enter the code.
+1.  Enter your primary phone number. This is the last step for identity verification.
+    * If you do not have a U.S. phone number or cannot verify by phone, click on “ "Verify your address by mail instead.” You will have to wait to receive a letter in the mail and then follow the instructions to enter the code.
```

Comment:
Minor: An extra set of double quotes got in here by the looks of it.

```suggestion
    * If you do not have a U.S. phone number or cannot verify by phone, click on “Verify your address by mail instead.” You will have to wait to receive a letter in the mail and then follow the instructions to enter the code.
```

Reply:
Got it. opened a new PR